TodoApp/test/TodoApp.Api.Tests/Controllers/ItemsControllerTests.cs

```diff
@@ -76,26 +91,62 @@ public void SetUp()
             Assert.That(item, Is.EqualTo(_items[0]).UsingItemEqualityComparer());
         }
 
+        [Test]
+        public async Task GetAsync_EmptyId_ReturnsBadRequest()
+        {
+            _controller.ModelState.Clear();
```

Comment:
You shouldn't be needing this since you create a new controller instance before each test (or case from a test case source)